packages/frontend/src/components/UnderlyingData/UnderlyingDataResultsTable.tsx

```diff
@@ -19,7 +20,17 @@ const UnderlyingDataResultsTable: FC<{
     resultsData: ApiQueryResults | undefined;
     isLoading: boolean;
     hasJoins?: boolean;
-}> = ({ fieldsMap, resultsData, isLoading, hasJoins }) => {
+    sortByUnderlyingValues: (
+        columnA: TableColumn,
+        columnB: TableColumn,
+    ) => number;
+}> = ({
+    fieldsMap,
+    resultsData,
+    isLoading,
+    hasJoins,
+    sortByUnderlyingValues,
```

Comment:
Should this function be called somewhere in this component ?

Reply:
yes of course, I reverted the wrong line...